support-frontend/assets/pages/supporter-plus-landing/setup/threeTierChecks.ts

```diff
@@ -41,3 +42,11 @@ export const threeTierCheckoutEnabled = (
 		countriesAffectedByVATStatus.includes(countryId)
 	);
 };
+
+export const additionalBenefitsEnabled = (
+	abParticipations: Participations,
+	countryGroupId: CountryGroupId,
+): boolean => {
+	const displayAdditionalBenefits = !!abParticipations.additionalBenefits;
+	return countryGroupId === 'UnitedStates' && displayAdditionalBenefits;
```

Comment:
Why can't this just live in the `canRun`?

Reply:
This was not required after I refactored